.changeset/eleven-cameras-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/preact': major
```

Comment:
Why is this a major? What did we break for the users? If we changed something, we should really explain to the users what we changed

Reply:
Thanks! I'll update it. It should be a patch